src/shogun/machine/gp/InferenceMethod.cpp

```diff
@@ -75,6 +75,7 @@ void CInferenceMethod::init()
 	SG_ADD((CSGObject**)&m_mean, "mean_function", "Mean function", MS_AVAILABLE);
 	SG_ADD((CSGObject**)&m_labels, "labels", "Labels", MS_NOT_AVAILABLE);
 	SG_ADD((CSGObject**)&m_features, "features", "Features", MS_NOT_AVAILABLE);
+	SG_ADD(&m_is_compute_gradients, "is_compute_gradients", "whether gradients are computed", MS_NOT_AVAILABLE);
```

Comment:
why is the field called "is" ?

what about "m_compute_gradients?

Minor "Whether gradients are computed" (capital W)